Cql/CodeGeneration.NET/Visitors/SimplifyExpressionsVisitor.cs

```diff
@@ -18,16 +18,17 @@ namespace Hl7.Cql.CodeGeneration.NET.Visitors
     /// <summary>
     /// This Visitor will (in most cases) create a new variable for a nested expression, 
     /// and assign the visited node's expression to that variable, thus unwinding the deeply nested
-    /// structure of Linq.Expression.
-    /// e.g. exprA(exprB(4)) will be turned into
+    /// structure of Linq.Expression to make it easier to debug and inspect the intermediate values.
+    ///
+    /// E.g. exprA(exprB(4)) will be turned into
     ///     var b = exprB(4)
     ///     var a = exprA(b)
     ///     return a;
     /// </summary>
     internal class SimplifyExpressionsVisitor : ExpressionVisitor
     {
-        private bool _atRoot = true;
-        private readonly List<BinaryExpression> _assignments = new();
+        private readonly List<BinaryExpression> _assignments = [];
```

Comment:
Keep assignments inside the constructor

Reply:
I totally disagree.

Reply:
Ok, but that's not true for these simple expression subclasses that have little state and no constructor logic at all. The whole point of the new default constructors is removing boilerplate constructors for these cases. So, if you keep your classes simple (as you should), one should not be "punished" by still having constructs that are necessary for classes that are too big.